README.md

```diff
@@ -82,6 +82,14 @@ An easy way to check and see which issues or pull requests will initially be mar
 
 To avoid triggering abuse prevention mechanisms on GitHub, only 30 issues and pull requests will be marked or closed per hour. If your repository has more than that, it will just take a few hours or days to mark them all.
 
+## Is closing stale issues really a good idea?
+
+In an ideal world with infinite resources, there would be no need for this app.
```

Comment:
I don't think that this is necessary - it feels a little defensive (which I understand).

Reply:
fwiw this statement really connects with me -- i saw @bkeepers mention it elsewhere, and it humanizes the effort